test/CacheManager.Tests/RedisTests.cs

```diff
@@ -1408,6 +1408,73 @@ public void Redis_ValueConverter_ValidateValuesTypesNotUsingSerializer<T>(T valu
             triggerResult.Should().BeTrue("Event should get triggered through the backplane.");
             eventTriggeredLocal.Should().Be(expectedRemoteTriggers, "Local cache event should be triggered one time");
         }
+
+        [Fact]
+        public void Redis_GetAllKeys()
```

Comment:
regarding unit tests in general. There must be tests for every combination of pattern matching for EVERY cache handle. 
Best is to at least have a set of tests which uses 
```
[Theory]
[ClassData(typeof(TestCacheManagers))]
```
(just look for that and you'll find a lot of tests) this will run tests with many combinations of CacheManager configurations).

A lot to do here before I can call that stable enough ;)